packages/ts/react-crud/src/autocrud.tsx

```diff
@@ -214,10 +212,12 @@ export function AutoCrud<TModel extends AbstractModel>({
       ) : (
         <SplitLayout theme="small">
           {mainSection}
-          <div className="auto-crud-form">
-            <div className="auto-crud-form-header">{formHeader}</div>
-            {autoForm}
-          </div>
+          {(!noNewButton || (item && item !== emptyItem)) && (
```

Comment:
```suggestion
          {(!noNewButton || (item !== emptyItem)) && (
```
I assume we don't need to check the item to be `undefined` as `undefined` never equals to `emptyItem`.

UPD: Just noticed that there is a lot of the same `item && item !== emptyItem` comparisons. Yeah, they are also redundant but we can address it in another PR if you want.

Reply:
I didn't apply your suggestion directly as it was necessary to also remove parentheses.

Reply:
The logic is actually different: we want an item AND not empty. By simplifying, we're accepting everything different from empty, including null/undefined.